data/factory_user.go

```diff
@@ -61,10 +61,14 @@ func (f *userFactory) InContext(ctx context.Context) *userFactory {
 
 // Create returns the user struct, which is saved to the database.
 func (f *userFactory) Create() (*ent.User, error) {
+	pw, err := bcrypt.GenerateFromPassword([]byte(f.Password), bcrypt.MinCost)
```

Comment:
It was, this is just for testing.

I use this if I want to create a model but don't care about the details. I can fill in the properties that I want and it will fill in the rest for me with some semi-random data. But even for testing I want to encrypt the password because else I can't test the authentication.